include/yaml-cpp/node/impl.h

```diff
@@ -368,24 +374,45 @@ inline typename to_value_t<T>::return_type to_value(const T& t) {
 }
 }
 
+template<typename Key, bool Streamable>
+struct key_to_string_impl {
```

Comment:
This is more of a streamable_to_string, right? I think it should go in the traits file instead, and rename.